# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/nvi/no-snprintf.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./PORT/clib/snprintf.c.orig	Wed Mar 21 12:42:09 2001
+++ ./PORT/clib/snprintf.c	Wed Mar 21 12:43:28 2001
@@ -1,3 +1,6 @@
+/* This is not needed. - Clifford */
+#if 0
+
 #include <sys/types.h>
 #include <sys/cdefs.h>
 
@@ -52,3 +55,5 @@
 	return (vsprintf(str, fmt, ap));
 #endif
 }
+
+#endif
